src/main/java/pl/pandziak/ticketbookingapp/domain/reservation/Reservation.java

```diff
@@ -0,0 +1,126 @@
+package pl.pandziak.ticketbookingapp.domain.reservation;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public final class Reservation {
```

Comment:
Czemu nie Lombok?

Reply:
Nie jestem za dobrze zaznajomiona z tą biblioteką. Ale postaram się to naprawić. :)